options:
  is_optitrack_on: true
  is_orientation_track_on: true

  filter_gain: 0.2

control:
    dsGainPos: 4.0
    lambda0Pos: 30.0
    lambda1Pos: 30.0
    dsGainOri: 4.0
    lambda0Ori: 5.0
    lambda1Ori: 2.75
    mass_ee: 0.01

target:
    pos: [0.5, 0.0, 0.2]
    quat: [0.7071068, 0.0, 0.7071068,  0.0]

leader:
    pos: [1.3, 0.0, 0.0]